prototype/core/impl/src/main/java/org/eclipse/sensinact/prototype/impl/snapshot/package-info.java

```diff
@@ -0,0 +1,14 @@
+/*********************************************************************
+* Copyright (c) 2022 Contributors to the Eclipse Foundation.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Eclipse Public License 2.0 which is available at
+* https://www.eclipse.org/legal/epl-2.0/
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors: Kentyou - initial implementation
+**********************************************************************/
+@org.osgi.annotation.bundle.Export
+@org.osgi.annotation.versioning.Version("0.2.0")
+package org.eclipse.sensinact.prototype.impl.snapshot;
```

Comment:
We should definitely not be exporting the implementation package

Reply:
Definitely a debugging relic